test/java/seedu/addressbook/common/UtilsTest.java

```diff
@@ -43,4 +43,33 @@ private void assertAreUnique(Object... objects) {
     private void assertNotUnique(Object... objects) {
         assertFalse(Utils.elementsAreUnique(Arrays.asList(objects)));
     }
+
+    @Test
+    public void isAnyNull() throws Exception {
+        // no null
+        assertNoNull();
+        assertNoNull(1);
+        assertNoNull("");
+        assertNoNull("abc");
+        assertNoNull("abc", "ab", "a");
+        assertNoNull(1, 2);
+        assertNoNull("abc", "abc");
+        assertNoNull("abc", "", "abc", "ABC");
+        assertNoNull("", "abc", "a", "abc");
+        assertNoNull(1, Integer.valueOf(1));
+
+        // has null
+        assertHasNull((Object) null);
+        assertHasNull(null, 1, Integer.valueOf(1));
+        assertHasNull(null, null);
+        assertHasNull(null, "a", "b", null);
+    }
```

Comment:
Good job for attempting the LO!

Good that you tested both the True and False cases. As mentioned in tutorial, you might also want to consider testing different datatypes together. 

It's also good to see that you split the tests nicely into 2 categories.

Keep it up!